packages/cli/src/Init.ts

```diff
@@ -26,7 +26,6 @@ export const defaultSchema = (provider: ConnectorType = 'postgresql') => {
 
 generator client {
   provider        = "prisma-client-js"
-  previewFeatures = ["mongoDb"]
 }
 
 datasource db {
```

Comment:
This if/else can go away completely I think. But maybe keep the code around and just replace `mongodb` wiht `future-new-connector` - we will need ths a few more times I guess.